indicators/views/views_program.py

```diff
@@ -285,26 +288,197 @@ def programs_rollup_export_csv(request):
 
 class BulkImportIndicatorsView(LoginRequiredMixin, UserPassesTestMixin, AccessMixin, View):
     """Returns bulk import .xlsx file"""
-
     redirect_field_name = None
 
+    COLUMNS = [
+        {"name": "Level", "required": True},
+        {"name": "No.", "required": True},
+        {"name": "Indicator", "required": True, 'field_name': 'name'},
+        {"name": "Sector", "required": False, 'field_name': 'sector'},
+        {"name": "Source", "required": False, 'field_name': 'source'},
+        {"name": "Definition", "required": False, 'field_name': 'definition'},
+        {"name": "Rationale or justification for indicator", "required": False, 'field_name': 'rationale'},
+        {"name": "Unit of measure", "required": True, 'field_name': 'unit_of_measure'},
+        {"name": "Number (#) or percentage (%)", "required": True, 'validation': 'uom_type_validation', 'field_name': 'uom_type'},
+        {"name": "Rationale for target", "required": False, 'field_name': 'rationale_for_target'},
+        {"name": "Baseline", "required": True, 'field_name': 'baseline'},
+        {"name": "Direction of change", "required": False, 'validation': 'dir_change_validation', 'field_name': 'direction_of_change'},
+        {"name": "Target frequency", "required": True, 'field_name': 'target_frequency'},
+        {"name": "Means of verification / data source", "required": False, 'field_name': 'means_of_verification'},
+        {"name": "Data collection method", "required": False, 'field_name': 'data_collection_method'},
+        {"name": "Data points", "required": False, 'field_name': 'data_points'},
+        {"name": "Responsible person(s) and team", "required": False, 'field_name': 'responsible_person'},
+        {"name": "Method of analysis", "required": False, 'field_name': 'method_of_analysis'},
+        {"name": "Information use", "required": False, 'field_name': 'information_use'},
+        {"name": "Data quality assurance details", "required": False, 'field_name': 'quality_assurance'},
+        {"name": "Data issues", "required": False, 'field_name': 'data_issues'},
+        {"name": "Comments", "required": False, 'field_name': 'comments'}
+    ]
+
+    title_style = NamedStyle('title_style')
+    title_style.font = Font(name='Calibri', size=18)
+    title_style.protection = Protection(locked=True)
+
+    required_header_style = NamedStyle('required_header_style')
+    required_header_style.font = Font(name='Calibri', size=11, color='FFFFFFFF')
+    required_header_style.fill = PatternFill('solid', fgColor='00000000')
+    required_header_style.protection = Protection(locked=True)
+
+    optional_header_style = NamedStyle('optional_header_style')
+    optional_header_style.font = Font(name='Calibri', size=11, color='00000000')
+    optional_header_style.fill = PatternFill('solid', fgColor='FFF2F2F2')
```

Comment:
In our primitive scss file, the closest color we use is F5F5F5.

Reply:
Thanks, I should have looked at that before :-)